### ISSUE TITLE
Fix traffic view on bond interface with Mofed 5.7

### ISSUE THREAD

**Issue by MrBr-github** [MrBr-github/lshca#76]
```
r.........ty01:~# lshca -w traffic
------------------------------------------------------------
Dev #1
 Desc: Mellanox Technologies MT2892 Family [ConnectX-6 Dx]
 PN: MCX621102AN-ADAT  rev. A5
 PSID: MT_0000000356
 SN: MT2...........
 FW: 22.34.1002
 Driver: mlnx_ofed-5.7-1.0.2.0
------------------------------------------------------------
 RDMA  |     Net     | TX_bps | RX_bps | PktSeqErr
------------------------------------------------------------
mlx5_0 |    bond0    |  0.0   |  0.0   |    0.0
       | enp3s0f0np0 |  0.0   |  0.0   |    0.0
mlx5_1 |    bond0    |  0.0   |  0.0   |    0.0
mlx5_1 | enp3s0f1np1 |  0.0   |  0.0   |    0.0
------------------------------------------------------------
```

**Comment by MrBr-github** [comment]
Same issue with IB, ROCE and DPU views

**Comment by MrBr-github** [comment]
Issue opened by mistake
Root cause: wrong bond configuration
Will be fixed by #77
